extensions/indexes/range/src/org/exist/indexing/range/RangeIndexConfig.java

```diff
@@ -96,8 +97,18 @@ private void parseChildren(NodeList configNodes, Map<String, String> namespaces)
                     } else {
                         idxConf.add(newConfig);
                     }
-                } catch (DatabaseConfigurationException e) {
-                    LOG.error("Invalid range index configuration: " + e.getMessage());
+                } catch (final DatabaseConfigurationException e) {
+                    String uri = null;
+                    final Document doc = node.getOwnerDocument();
+                    if(doc != null) {
+                        uri = doc.getDocumentURI();
+                    }
+
+                    if(uri != null) {
```

Comment:
better style is `if (a==null) then {b} else {c}` in stead of `if (a!=null) then {c} else {b}` . But leave as is, as it is a minor-minor remark.